02-Context-or-Narrative/State-of-PureScript-Documentation-2019.md

```diff
@@ -183,15 +82,17 @@ Here's what has been added to the above since then:
 | A Guide to the PureScript Numeric Hierarchy | Explanation | New Learners | Read the Docs
 | Make the Leap from JavaScript to PureScript | Getting Started + Explanation | Javascript developers | Blog Post series
 
```

Comment:
In your version (#65), you included the table of who had write/deploy privilege to PS website. I didn't include that here because I thought people might not read through the entire document and just assume people were lazy rather than being burned out.

Reply:
It's ok to leave it out for now. I suspect it might be useful information which could affect the outcomes, so I might propose to add it back in later, but I'd have to think about it more.